core/state/journal.go

```diff
@@ -95,8 +141,83 @@ func (j *journal) copy() *journal {
 		entries = append(entries, j.entries[i].copy())
 	}
 	return &journal{
-		entries: entries,
-		dirties: maps.Clone(j.dirties),
+		entries:        entries,
+		dirties:        maps.Clone(j.dirties),
+		validRevisions: slices.Clone(j.validRevisions),
+		nextRevisionId: j.nextRevisionId,
+	}
+}
+
+func (j *journal) JournalAccessListAddAccount(addr common.Address) {
```

Comment:
Just a quick note, these names all have `Journal` as a prefix but it's not needed.

Reply:
Fixed and rebased